src/auth/auth.service.ts

```
@@ -304,5 +310,35 @@ export class AuthService {
       }
     };
   }
+
+  async suspendUser(adminId: string, targetUserId: string): Promise<User> {
+    // Get the admin user
+    const admin = await this.userRepository.findOne({ where: { id: adminId } });
+    if (!admin || (admin.role !== UserRole.ADMIN && admin.role !== UserRole.SUPER_ADMIN)) {
```

ESLint

[error] Replace `!admin·||·(admin.role·!==·UserRole.ADMIN·&&·admin.role·!==·UserRole.SUPER_ADMIN)` with `⏎······!admin·||⏎······(admin.role·!==·UserRole.ADMIN·&&·admin.role·!==·UserRole.SUPER_ADMIN)⏎····`

(prettier/prettier)